On March 6th, Governor Brad Little (ID-R) signed [House Bill 40](https://legislature.idaho.gov/sessioninfo/2025/legislation/H0040/), lowering the state's top marginal income tax rate from 5.695% to 5.3%. A week later, he signed [House Bill 231](https://legislature.idaho.gov/sessioninfo/2025/legislation/h0231/), which raised the state’s grocery tax credit from $120 (or $140 for filers aged 65 or older) to $155.

We at PolicyEngine have updated our model to reflect these changes and analyzed their effects on the state of Idaho and its residents.


Key results for 2025:

* Reduces state revenues by $225 million
* Benefits 87% of Idaho residents
* Has no effect on the Supplemental Poverty Measure 
* Increase the Gini index of inequality by 0.01%

*Use PolicyEngine to view the [full results](https://policyengine.org/us/policy?focus=policyOutput.policyBreakdown&reform=2&region=id&timePeriod=2025&baseline=80392) or calculate the [effect on your household](https://policyengine.org/us/household?focus=intro&reform=2&region=id&timePeriod=2025&baseline=80392).*

In [2]:
from IPython.display import Markdown
import pandas as pd
import plotly.express as px
import plotly.graph_objects as go


# Define PolicyEngine's color palette
BLACK = "#000000"
BLUE_LIGHT = "#D8E6F3"
BLUE_PRIMARY = "#2C6496"
DARK_BLUE_HOVER = "#1d3e5e"
DARK_GRAY = "#616161"
DARKEST_BLUE = "#0C1A27"
GRAY = "#808080"
LIGHT_GRAY = "#F2F2F2"
MEDIUM_DARK_GRAY = "#D2D2D2"
MEDIUM_LIGHT_GRAY = "#BDBDBD"
WHITE = "#FFFFFF"

## Tax Reforms

Idaho has undergone several tax reforms in recent years. However, in 2024, Idaho has two tax brackets: one at 0% and another at 5.695%. For single filers, the 5.695% rate does not apply until their taxable income exceeds $4,673. House Bill 40 reduces the 5.695% rate to 5.3%. 

In 2025, using inflation projections from the Congressional Budget Office, this new rate would apply when a single filer’s taxable income reaches $4,777. Table 1 shows when the new marginal tax rate would apply based on filing status. 

In [3]:
# Create a table for the household impacts
filing_status = ["Head of Household, Joint, and Surviving Spouse", "Single and Separate"]
income_threshold = [9553, 4777]

def add_thousands_separator(x):
    return "{:,}".format(x)

df = pd.DataFrame({
    "Filing Status": filing_status,
    "Taxable Income Threshold ($)": income_threshold
})

df["Filing Status"] = df["Filing Status"]
df["Taxable Income Threshold ($)"] = df["Taxable Income Threshold ($)"].apply(add_thousands_separator)

Markdown("**Table 1: Taxable Income Where Idaho’s Top Marginal Tax Rate Applies Based on Filing Status in 2025**\n\n" + df.to_markdown(index=False))

**Table 1: Taxable Income Where Idaho’s Top Marginal Tax Rate Applies Based on Filing Status in 2025**

| Filing Status                                  | Taxable Income Threshold ($)   |
|:-----------------------------------------------|:-------------------------------|
| Head of Household, Joint, and Surviving Spouse | 9,553                          |
| Single and Separate                            | 4,777                          |

The grocery tax credit provides a fully refundable credit to households based on the number of family members. The grocery tax credit is not subject to any income limits or phase-outs, meaning any household in Idaho can claim the credit. 

The only exception is if a household receives benefits for the Supplemental Nutrition Assistance Program (SNAP)  throughout the year, they do not qualify for the credit. In 2024, the credit’s value was $120 per member, with an additional $20 benefit for claimants 65 or over. House Bill 231 raises the credit value to $155 while eliminating the $20 supplement for elderly household members. 

## Household Impacts

The two examined income tax changes have different effects on households based on their composition and earnings. Here we consider some examples, displayed in Table 2.

A childless adult with no earnings would be unaffected by either tax provision. Since single filers in Idaho can take a standard deduction of $14,600 and the top marginal tax rate does not go into effect until they have $4,673 of taxable income, a childless adult must earn $19,273 before their net income would be affected by the rate change. Additionally, since a childless adult with no earnings, but who meets [SNAP’s work requirements](https://www.fns.usda.gov/snap/work-requirements) (ex., is participating in a work program at least 80 hours per month), is eligible for SNAP benefits, the increase in grocery tax credit does not change their net income either. They would have to earn over $20,000 to no longer qualify for SNAP and therefore be eligible for the grocery tax credit. 

A single parent with two kids earning $50,000 annually would gain $176. As their income excludes them from SNAP benefits, the household qualifies for the grocery tax credit, which increases by $35 per household member for a total of $105. Since the household's taxable income is subject to the reduced top marginal tax rate, their tax liability is reduced by $71.

Finally, an elderly couple with no children and $200,000 of annual earnings would see their net income increase by $214. Since each spouse was scheduled to receive $140 in grocery credits ($120 plus the $20 aged amount) and will now receive $155 each, their total allotment increases by $30. Additionally, the top marginal rate cut lowers their state income tax liability by $184. 

In [4]:
# Create a table for the household impacts
household_composition = ["Single, No Children, No Earnings", "Single, Two Children, $50,000", "Married (Elderly), No Children, $200,000"]
income_change = [0, 176, 214]

def add_thousands_separator(x):
    return "{:,}".format(x)

df = pd.DataFrame({
    "Household Composition": household_composition,
    "Change in Net Income ($)": income_change
})

df["Household Composition"] = df["Household Composition"]
df["Change in Net Income ($)"] = df["Change in Net Income ($)"].apply(add_thousands_separator)

Markdown("**Table 2: Change in Net Income Based on Household Composition in 2025**\n\n" + df.to_markdown(index=False))

**Table 2: Change in Net Income Based on Household Composition in 2025**

| Household Composition                    |   Change in Net Income ($) |
|:-----------------------------------------|---------------------------:|
| Single, No Children, No Earnings         |                          0 |
| Single, Two Children, $50,000            |                        176 |
| Married (Elderly), No Children, $200,000 |                        214 |

The figure below displays how a single parent of two’s net income changes as household earnings vary for tax year 2025. 

In [16]:
employment_income = [0, 34000, 35000, 39000, 39500, 40000, 40500, 41000, 41500, 42000, 42500, 43000, 43500, 44000, 44500, 45000, 45500, 46000, 46500, 47000, 47500, 48000, 48500, 49000, 49500, 50000, 50500, 51000, 51500, 52000, 52500, 53000, 53500, 54000, 54500, 55000, 55500, 56000, 56500, 57000, 57500, 58000, 58500, 59000, 59500, 60000, 60500, 61000, 61500, 62000, 62500, 63000, 63500, 64000, 64500, 65000, 65500, 66000, 66500, 67000, 67500, 68000, 68500, 69000, 69500, 70000, 70500, 71000, 71500, 72000, 72500, 73000, 73500, 74000, 74500, 75000, 75500, 76000, 76500, 77000, 77500, 78000, 78500, 79000, 79500, 80000, 80500, 81000, 81500, 82000, 82500, 83000, 83500, 84000, 84500, 85000, 85500, 86000, 86500, 87000, 87500, 88000, 88500, 89000, 89500, 90000, 90500, 91000, 91500, 92000, 92500, 93000, 93500, 94000, 94500, 95000, 95500, 96000, 96500, 97000, 97500, 98000, 98500, 99000, 99500, 100000, 100500, 101000, 101500, 102000, 102500, 103000, 103500, 104000, 104500, 105000, 105500, 106000, 106500, 107000, 107500, 108000, 108500, 109000, 109500, 110000, 110500, 111000, 111500, 112000, 112500, 113000, 113500, 114000, 114500, 115000, 115500, 116000, 116500, 117000, 117500, 118000, 118500, 119000, 119500, 120000, 120500, 121000, 121500, 122000, 122500, 123000, 123500, 124000, 124500, 125000, 125500, 126000, 126500, 127000, 127500, 128000, 128500, 129000, 129500, 130000, 130500, 131000, 131500, 132000, 132500, 133000, 133500, 134000, 134500, 135000, 135500, 136000, 136500, 137000, 137500, 138000, 138500, 139000, 139500, 140000, 140500, 141000, 141500, 142000, 142500, 143000, 143500, 144000, 144500, 145000, 145500, 146000, 146500, 147000, 147500, 148000, 148500, 149000, 149500, 150000, 150500, 151000, 151500, 152000, 152500, 153000, 153500, 154000, 154500, 155000, 155500, 156000, 156500, 157000, 157500, 158000, 158500, 159000, 159500, 160000, 160500, 161000, 161500, 162000, 162500, 163000, 163500, 164000, 164500, 165000, 165500, 166000, 166500, 167000, 167500, 168000, 168500, 169000, 169500, 170000, 170500, 171000, 171500, 172000, 172500, 173000, 173500, 174000, 174500, 175000, 175500, 176000, 176500, 177000, 177500, 178000, 178500, 179000, 179500, 180000, 180500, 181000, 181500, 182000, 182500, 183000, 183500, 184000, 184500, 185000, 185500, 186000, 186500, 187000, 187500, 188000, 188500, 189000, 189500, 190000, 190500, 191000, 191500, 192000, 192500, 193000, 193500, 194000, 194500, 195000, 195500, 196000, 196500, 197000, 197500, 198000, 198500, 199000, 199500, 200000]

net_income_change = [0, 0, 105, 105, 119, 136, 138, 140, 142, 144, 146, 148, 150, 152, 154, 156, 158, 160, 162, 164, 166, 167.97, 169.93, 171.9, 173.87, 175.84, 177.8, 179.77, 181.74, 183.7, 185.67, 187.64, 189.61, 191.57, 193.54, 195.51, 197.48, 199.44, 201.41, 203.38, 205.34, 207.31, 209.28, 211.25, 213.21, 215.18, 217.15, 219.11, 221.08, 223.05, 225.02, 226.98, 228.95, 230.92, 232.89, 234.85, 236.82, 238.79, 240.75, 242.72, 244.69, 246.66, 248.62, 250.59, 252.56, 254.52, 256.49, 258.46, 260.43, 262.39, 264.36, 266.33, 268.3, 270.26, 272.23, 274.2, 276.16, 278.13, 280.1, 282.07, 284.03, 286, 287.97, 289.93, 291.9, 293.87, 295.84, 297.8, 299.77, 301.74, 303.7, 305.67, 307.64, 309.61, 311.57, 313.54, 315.51, 317.48, 319.44, 321.41, 323.38, 325.34, 327.31, 329.28, 331.25, 333.21, 335.18, 337.15, 339.11, 341.08, 343.05, 345.02, 346.98, 348.95, 350.92, 352.89, 354.85, 356.82, 358.79, 360.75, 362.72, 364.69, 366.66, 368.62, 370.59, 372.56, 374.52, 376.49, 378.46, 380.43, 382.39, 384.36, 386.33, 388.3, 390.26, 392.23, 394.2, 396.16, 398.13, 400.1, 402.07, 404.03, 406, 407.97, 409.93, 411.9, 413.87, 415.84, 417.8, 419.77, 421.74, 423.7, 425.67, 427.64, 429.61, 431.57, 433.54, 435.51, 437.48, 439.44, 441.41, 443.38, 445.34, 447.31, 449.28, 451.25, 453.21, 455.18, 457.15, 459.11, 461.08, 463.05, 465.02, 466.98, 468.95, 470.92, 472.89, 474.85, 476.82, 478.79, 480.75, 482.72, 484.69, 486.66, 488.62, 490.59, 492.56, 494.52, 496.49, 498.46, 500.43, 502.39, 504.36, 506.33, 508.3, 510.26, 512.23, 514.2, 516.16, 518.13, 520.1, 522.07, 524.03, 526, 527.97, 529.93, 531.9, 533.87, 535.84, 537.8, 539.77, 541.74, 543.7, 545.67, 547.64, 549.61, 551.57, 553.54, 555.51, 557.48, 559.44, 561.41, 563.38, 565.34, 567.31, 569.28, 571.25, 573.21, 575.18, 577.15, 579.11, 581.08, 583.05, 585.02, 586.98, 588.95, 590.92, 592.89, 594.85, 596.82, 598.79, 600.75, 602.72, 604.69, 606.66, 608.62, 610.59, 612.56, 614.52, 616.49, 618.46, 620.43, 622.39, 624.36, 626.33, 628.3, 630.26, 632.23, 634.2, 636.16, 638.13, 640.1, 642.07, 644.03, 646, 647.97, 649.93, 651.9, 653.87, 655.84, 657.8, 659.77, 661.74, 663.7, 665.67, 667.64, 669.61, 671.57, 673.54, 675.51, 677.48, 679.44, 681.41, 683.38, 685.34, 687.31, 689.28, 691.25, 693.21, 695.18, 697.15, 699.11, 701.08, 703.05, 705.02, 706.98, 708.95, 710.92, 712.89, 714.85, 716.82, 718.79, 720.75, 722.72, 724.69, 726.66, 728.62, 730.59, 732.56, 734.52, 736.49, 738.46, 740.43, 742.39, 744.36, 746.33, 748.3, 750.26, 752.23, 754.2, 756.16, 758.13, 760.1, 762.07, 764.03, 766]


df = pd.DataFrame({
    "Employment Income": employment_income,
    "Change in net income": net_income_change
})

fig = px.line(
    df,
    x="Employment Income",
    y="Change in net income",
    color_discrete_sequence=[BLUE_PRIMARY],
    title="Change in Household Net Income by Employment Income"
).update_layout(
    font=dict(family="Roboto Serif"),
    xaxis_title="Household head employment income ($)",
    yaxis_title="Change in net income ($)",
    xaxis_tickformat=",",
    yaxis_tickformat=",",
    font_color=BLACK,
    margin={
        "l": 50,
        "r": 50,
        "b": 100,
        "t": 100,
        "pad": 4
    },
    annotations=[
        {
            "x": 1,
            "y": -0.25,
            "xref": "paper",
            "yref": "paper",
            "text": "Source: PolicyEngine US",
            "showarrow": False,
            "font": {
                "family": "Roboto Serif",
                "size": 10,
                "color": DARK_GRAY
            }
        }
    ],
    images=[
        {
            "source": "/logo512.png",
            "x": 1,
            "y": -0.2,
            "xref": "paper",
            "yref": "paper",
            "sizex": 0.1,
            "sizey": 0.1,
            "xanchor": "right",
            "yanchor": "bottom"
        }
    ],
)

Markdown("**Figure 1: Change in Net Income for a Single Parent of Two Children in 2025**\n\n" + f"```plotly\n{fig.to_json()}\n```")

**Figure 1: Change in Net Income for a Single Parent of Two Children in 2025**

```plotly
{"data":[{"hovertemplate":"Employment Income=%{x}\u003cbr\u003eChange in net income=%{y}\u003cextra\u003e\u003c\u002fextra\u003e","legendgroup":"","line":{"color":"#2C6496","dash":"solid"},"marker":{"symbol":"circle"},"mode":"lines","name":"","orientation":"v","showlegend":false,"x":[0,34000,35000,39000,39500,40000,40500,41000,41500,42000,42500,43000,43500,44000,44500,45000,45500,46000,46500,47000,47500,48000,48500,49000,49500,50000,50500,51000,51500,52000,52500,53000,53500,54000,54500,55000,55500,56000,56500,57000,57500,58000,58500,59000,59500,60000,60500,61000,61500,62000,62500,63000,63500,64000,64500,65000,65500,66000,66500,67000,67500,68000,68500,69000,69500,70000,70500,71000,71500,72000,72500,73000,73500,74000,74500,75000,75500,76000,76500,77000,77500,78000,78500,79000,79500,80000,80500,81000,81500,82000,82500,83000,83500,84000,84500,85000,85500,86000,86500,87000,87500,88000,88500,89000,89500,90000,90500,91000,91500,92000,92500,93000,93500,94000,94500,95000,95500,96000,96500,97000,97500,98000,98500,99000,99500,100000,100500,101000,101500,102000,102500,103000,103500,104000,104500,105000,105500,106000,106500,107000,107500,108000,108500,109000,109500,110000,110500,111000,111500,112000,112500,113000,113500,114000,114500,115000,115500,116000,116500,117000,117500,118000,118500,119000,119500,120000,120500,121000,121500,122000,122500,123000,123500,124000,124500,125000,125500,126000,126500,127000,127500,128000,128500,129000,129500,130000,130500,131000,131500,132000,132500,133000,133500,134000,134500,135000,135500,136000,136500,137000,137500,138000,138500,139000,139500,140000,140500,141000,141500,142000,142500,143000,143500,144000,144500,145000,145500,146000,146500,147000,147500,148000,148500,149000,149500,150000,150500,151000,151500,152000,152500,153000,153500,154000,154500,155000,155500,156000,156500,157000,157500,158000,158500,159000,159500,160000,160500,161000,161500,162000,162500,163000,163500,164000,164500,165000,165500,166000,166500,167000,167500,168000,168500,169000,169500,170000,170500,171000,171500,172000,172500,173000,173500,174000,174500,175000,175500,176000,176500,177000,177500,178000,178500,179000,179500,180000,180500,181000,181500,182000,182500,183000,183500,184000,184500,185000,185500,186000,186500,187000,187500,188000,188500,189000,189500,190000,190500,191000,191500,192000,192500,193000,193500,194000,194500,195000,195500,196000,196500,197000,197500,198000,198500,199000,199500,200000],"xaxis":"x","y":[0.0,0.0,105.0,105.0,119.0,136.0,138.0,140.0,142.0,144.0,146.0,148.0,150.0,152.0,154.0,156.0,158.0,160.0,162.0,164.0,166.0,167.97,169.93,171.9,173.87,175.84,177.8,179.77,181.74,183.7,185.67,187.64,189.61,191.57,193.54,195.51,197.48,199.44,201.41,203.38,205.34,207.31,209.28,211.25,213.21,215.18,217.15,219.11,221.08,223.05,225.02,226.98,228.95,230.92,232.89,234.85,236.82,238.79,240.75,242.72,244.69,246.66,248.62,250.59,252.56,254.52,256.49,258.46,260.43,262.39,264.36,266.33,268.3,270.26,272.23,274.2,276.16,278.13,280.1,282.07,284.03,286.0,287.97,289.93,291.9,293.87,295.84,297.8,299.77,301.74,303.7,305.67,307.64,309.61,311.57,313.54,315.51,317.48,319.44,321.41,323.38,325.34,327.31,329.28,331.25,333.21,335.18,337.15,339.11,341.08,343.05,345.02,346.98,348.95,350.92,352.89,354.85,356.82,358.79,360.75,362.72,364.69,366.66,368.62,370.59,372.56,374.52,376.49,378.46,380.43,382.39,384.36,386.33,388.3,390.26,392.23,394.2,396.16,398.13,400.1,402.07,404.03,406.0,407.97,409.93,411.9,413.87,415.84,417.8,419.77,421.74,423.7,425.67,427.64,429.61,431.57,433.54,435.51,437.48,439.44,441.41,443.38,445.34,447.31,449.28,451.25,453.21,455.18,457.15,459.11,461.08,463.05,465.02,466.98,468.95,470.92,472.89,474.85,476.82,478.79,480.75,482.72,484.69,486.66,488.62,490.59,492.56,494.52,496.49,498.46,500.43,502.39,504.36,506.33,508.3,510.26,512.23,514.2,516.16,518.13,520.1,522.07,524.03,526.0,527.97,529.93,531.9,533.87,535.84,537.8,539.77,541.74,543.7,545.67,547.64,549.61,551.57,553.54,555.51,557.48,559.44,561.41,563.38,565.34,567.31,569.28,571.25,573.21,575.18,577.15,579.11,581.08,583.05,585.02,586.98,588.95,590.92,592.89,594.85,596.82,598.79,600.75,602.72,604.69,606.66,608.62,610.59,612.56,614.52,616.49,618.46,620.43,622.39,624.36,626.33,628.3,630.26,632.23,634.2,636.16,638.13,640.1,642.07,644.03,646.0,647.97,649.93,651.9,653.87,655.84,657.8,659.77,661.74,663.7,665.67,667.64,669.61,671.57,673.54,675.51,677.48,679.44,681.41,683.38,685.34,687.31,689.28,691.25,693.21,695.18,697.15,699.11,701.08,703.05,705.02,706.98,708.95,710.92,712.89,714.85,716.82,718.79,720.75,722.72,724.69,726.66,728.62,730.59,732.56,734.52,736.49,738.46,740.43,742.39,744.36,746.33,748.3,750.26,752.23,754.2,756.16,758.13,760.1,762.07,764.03,766.0],"yaxis":"y","type":"scatter"}],"layout":{"template":{"data":{"histogram2dcontour":[{"type":"histogram2dcontour","colorbar":{"outlinewidth":0,"ticks":""},"colorscale":[[0.0,"#0d0887"],[0.1111111111111111,"#46039f"],[0.2222222222222222,"#7201a8"],[0.3333333333333333,"#9c179e"],[0.4444444444444444,"#bd3786"],[0.5555555555555556,"#d8576b"],[0.6666666666666666,"#ed7953"],[0.7777777777777778,"#fb9f3a"],[0.8888888888888888,"#fdca26"],[1.0,"#f0f921"]]}],"choropleth":[{"type":"choropleth","colorbar":{"outlinewidth":0,"ticks":""}}],"histogram2d":[{"type":"histogram2d","colorbar":{"outlinewidth":0,"ticks":""},"colorscale":[[0.0,"#0d0887"],[0.1111111111111111,"#46039f"],[0.2222222222222222,"#7201a8"],[0.3333333333333333,"#9c179e"],[0.4444444444444444,"#bd3786"],[0.5555555555555556,"#d8576b"],[0.6666666666666666,"#ed7953"],[0.7777777777777778,"#fb9f3a"],[0.8888888888888888,"#fdca26"],[1.0,"#f0f921"]]}],"heatmap":[{"type":"heatmap","colorbar":{"outlinewidth":0,"ticks":""},"colorscale":[[0.0,"#0d0887"],[0.1111111111111111,"#46039f"],[0.2222222222222222,"#7201a8"],[0.3333333333333333,"#9c179e"],[0.4444444444444444,"#bd3786"],[0.5555555555555556,"#d8576b"],[0.6666666666666666,"#ed7953"],[0.7777777777777778,"#fb9f3a"],[0.8888888888888888,"#fdca26"],[1.0,"#f0f921"]]}],"heatmapgl":[{"type":"heatmapgl","colorbar":{"outlinewidth":0,"ticks":""},"colorscale":[[0.0,"#0d0887"],[0.1111111111111111,"#46039f"],[0.2222222222222222,"#7201a8"],[0.3333333333333333,"#9c179e"],[0.4444444444444444,"#bd3786"],[0.5555555555555556,"#d8576b"],[0.6666666666666666,"#ed7953"],[0.7777777777777778,"#fb9f3a"],[0.8888888888888888,"#fdca26"],[1.0,"#f0f921"]]}],"contourcarpet":[{"type":"contourcarpet","colorbar":{"outlinewidth":0,"ticks":""}}],"contour":[{"type":"contour","colorbar":{"outlinewidth":0,"ticks":""},"colorscale":[[0.0,"#0d0887"],[0.1111111111111111,"#46039f"],[0.2222222222222222,"#7201a8"],[0.3333333333333333,"#9c179e"],[0.4444444444444444,"#bd3786"],[0.5555555555555556,"#d8576b"],[0.6666666666666666,"#ed7953"],[0.7777777777777778,"#fb9f3a"],[0.8888888888888888,"#fdca26"],[1.0,"#f0f921"]]}],"surface":[{"type":"surface","colorbar":{"outlinewidth":0,"ticks":""},"colorscale":[[0.0,"#0d0887"],[0.1111111111111111,"#46039f"],[0.2222222222222222,"#7201a8"],[0.3333333333333333,"#9c179e"],[0.4444444444444444,"#bd3786"],[0.5555555555555556,"#d8576b"],[0.6666666666666666,"#ed7953"],[0.7777777777777778,"#fb9f3a"],[0.8888888888888888,"#fdca26"],[1.0,"#f0f921"]]}],"mesh3d":[{"type":"mesh3d","colorbar":{"outlinewidth":0,"ticks":""}}],"scatter":[{"fillpattern":{"fillmode":"overlay","size":10,"solidity":0.2},"type":"scatter"}],"parcoords":[{"type":"parcoords","line":{"colorbar":{"outlinewidth":0,"ticks":""}}}],"scatterpolargl":[{"type":"scatterpolargl","marker":{"colorbar":{"outlinewidth":0,"ticks":""}}}],"bar":[{"error_x":{"color":"#2a3f5f"},"error_y":{"color":"#2a3f5f"},"marker":{"line":{"color":"#E5ECF6","width":0.5},"pattern":{"fillmode":"overlay","size":10,"solidity":0.2}},"type":"bar"}],"scattergeo":[{"type":"scattergeo","marker":{"colorbar":{"outlinewidth":0,"ticks":""}}}],"scatterpolar":[{"type":"scatterpolar","marker":{"colorbar":{"outlinewidth":0,"ticks":""}}}],"histogram":[{"marker":{"pattern":{"fillmode":"overlay","size":10,"solidity":0.2}},"type":"histogram"}],"scattergl":[{"type":"scattergl","marker":{"colorbar":{"outlinewidth":0,"ticks":""}}}],"scatter3d":[{"type":"scatter3d","line":{"colorbar":{"outlinewidth":0,"ticks":""}},"marker":{"colorbar":{"outlinewidth":0,"ticks":""}}}],"scattermapbox":[{"type":"scattermapbox","marker":{"colorbar":{"outlinewidth":0,"ticks":""}}}],"scatterternary":[{"type":"scatterternary","marker":{"colorbar":{"outlinewidth":0,"ticks":""}}}],"scattercarpet":[{"type":"scattercarpet","marker":{"colorbar":{"outlinewidth":0,"ticks":""}}}],"carpet":[{"aaxis":{"endlinecolor":"#2a3f5f","gridcolor":"white","linecolor":"white","minorgridcolor":"white","startlinecolor":"#2a3f5f"},"baxis":{"endlinecolor":"#2a3f5f","gridcolor":"white","linecolor":"white","minorgridcolor":"white","startlinecolor":"#2a3f5f"},"type":"carpet"}],"table":[{"cells":{"fill":{"color":"#EBF0F8"},"line":{"color":"white"}},"header":{"fill":{"color":"#C8D4E3"},"line":{"color":"white"}},"type":"table"}],"barpolar":[{"marker":{"line":{"color":"#E5ECF6","width":0.5},"pattern":{"fillmode":"overlay","size":10,"solidity":0.2}},"type":"barpolar"}],"pie":[{"automargin":true,"type":"pie"}]},"layout":{"autotypenumbers":"strict","colorway":["#636efa","#EF553B","#00cc96","#ab63fa","#FFA15A","#19d3f3","#FF6692","#B6E880","#FF97FF","#FECB52"],"font":{"color":"#2a3f5f"},"hovermode":"closest","hoverlabel":{"align":"left"},"paper_bgcolor":"white","plot_bgcolor":"#E5ECF6","polar":{"bgcolor":"#E5ECF6","angularaxis":{"gridcolor":"white","linecolor":"white","ticks":""},"radialaxis":{"gridcolor":"white","linecolor":"white","ticks":""}},"ternary":{"bgcolor":"#E5ECF6","aaxis":{"gridcolor":"white","linecolor":"white","ticks":""},"baxis":{"gridcolor":"white","linecolor":"white","ticks":""},"caxis":{"gridcolor":"white","linecolor":"white","ticks":""}},"coloraxis":{"colorbar":{"outlinewidth":0,"ticks":""}},"colorscale":{"sequential":[[0.0,"#0d0887"],[0.1111111111111111,"#46039f"],[0.2222222222222222,"#7201a8"],[0.3333333333333333,"#9c179e"],[0.4444444444444444,"#bd3786"],[0.5555555555555556,"#d8576b"],[0.6666666666666666,"#ed7953"],[0.7777777777777778,"#fb9f3a"],[0.8888888888888888,"#fdca26"],[1.0,"#f0f921"]],"sequentialminus":[[0.0,"#0d0887"],[0.1111111111111111,"#46039f"],[0.2222222222222222,"#7201a8"],[0.3333333333333333,"#9c179e"],[0.4444444444444444,"#bd3786"],[0.5555555555555556,"#d8576b"],[0.6666666666666666,"#ed7953"],[0.7777777777777778,"#fb9f3a"],[0.8888888888888888,"#fdca26"],[1.0,"#f0f921"]],"diverging":[[0,"#8e0152"],[0.1,"#c51b7d"],[0.2,"#de77ae"],[0.3,"#f1b6da"],[0.4,"#fde0ef"],[0.5,"#f7f7f7"],[0.6,"#e6f5d0"],[0.7,"#b8e186"],[0.8,"#7fbc41"],[0.9,"#4d9221"],[1,"#276419"]]},"xaxis":{"gridcolor":"white","linecolor":"white","ticks":"","title":{"standoff":15},"zerolinecolor":"white","automargin":true,"zerolinewidth":2},"yaxis":{"gridcolor":"white","linecolor":"white","ticks":"","title":{"standoff":15},"zerolinecolor":"white","automargin":true,"zerolinewidth":2},"scene":{"xaxis":{"backgroundcolor":"#E5ECF6","gridcolor":"white","linecolor":"white","showbackground":true,"ticks":"","zerolinecolor":"white","gridwidth":2},"yaxis":{"backgroundcolor":"#E5ECF6","gridcolor":"white","linecolor":"white","showbackground":true,"ticks":"","zerolinecolor":"white","gridwidth":2},"zaxis":{"backgroundcolor":"#E5ECF6","gridcolor":"white","linecolor":"white","showbackground":true,"ticks":"","zerolinecolor":"white","gridwidth":2}},"shapedefaults":{"line":{"color":"#2a3f5f"}},"annotationdefaults":{"arrowcolor":"#2a3f5f","arrowhead":0,"arrowwidth":1},"geo":{"bgcolor":"white","landcolor":"#E5ECF6","subunitcolor":"white","showland":true,"showlakes":true,"lakecolor":"white"},"title":{"x":0.05},"mapbox":{"style":"light"}}},"xaxis":{"anchor":"y","domain":[0.0,1.0],"title":{"text":"Household head employment income ($)"},"tickformat":","},"yaxis":{"anchor":"x","domain":[0.0,1.0],"title":{"text":"Change in net income ($)"},"tickformat":","},"legend":{"tracegroupgap":0},"title":{"text":"Change in Household Net Income by Employment Income"},"font":{"family":"Roboto Serif","color":"#000000"},"margin":{"l":50,"r":50,"b":100,"t":100,"pad":4},"annotations":[{"font":{"color":"#616161","family":"Roboto Serif","size":10},"showarrow":false,"text":"Source: PolicyEngine US","x":1,"xref":"paper","y":-0.25,"yref":"paper"}],"images":[{"sizex":0.1,"sizey":0.1,"source":"\u002flogo512.png","x":1,"xanchor":"right","xref":"paper","y":-0.2,"yanchor":"bottom","yref":"paper"}]}}
```

## Statewide Impacts

Using PolicyEngine static modeling, lowering Idaho’s top marginal tax rate will reduce state revenues by [$170.4 million](https://policyengine.org/us/policy?focus=policyOutput.budgetaryImpact.overall&reform=2&region=id&timePeriod=2025&baseline=80396) for tax year 2025. Raising the value of the Idaho grocery tax credit will cost [$54.5 million](https://policyengine.org/us/policy?focus=policyOutput.policyBreakdown&reform=2&region=id&timePeriod=2025&baseline=80387). With both policies, the combined state budgetary impact is [$224.9 million](https://policyengine.org/us/policy?focus=policyOutput.budgetaryImpact.overall&reform=2&region=id&timePeriod=2025&baseline=80392&household=52252).

87% of residents in Idaho will see [an increase in their net income](https://policyengine.org/us/policy?focus=policyOutput.winnersAndLosers.incomeDecile&reform=2&region=id&timePeriod=2025&baseline=80392&household=52252). The range of residents being net beneficiaries varies based on their income decile. 46% of residents in the lowest income decile will see their net income increase, while 100% in the highest two deciles will see a gain. 

In [6]:
from plotly.subplots import make_subplots

# Income deciles
deciles = list(range(1, 11))

# Data for tax cut effects
gain_more_than_5pct = [2.4, 0, 0, 0, 0, 0, 0, 0, 0, 0]  # Dark blue
gain_less_than_5pct = [43, 71, 82, 81, 95, 96, 100, 99, 100, 100]  # Light blue
no_change = [54, 29, 18, 19, 5, 4, 0, 1, 0, 0]  # Neutral grey
loss_less_than_5pct = [0, 0, 0, 0, 0, 0, 0, 0, 0, 0]  # Light red (empty in this case)
loss_more_than_5pct = [0, 0, 0, 0, 0, 0, 0, 0, 0, 0]  # Dark red (empty in this case)

# Direct input values for the "All" category
all_gain_more = 0
all_gain_less = 87
all_no_change = 13
all_loss_less = 0
all_loss_more = 0

# Create labels for deciles
labels_deciles = [f"{i}" for i in deciles]

# Create DataFrame for deciles
df_deciles = pd.DataFrame({
    "Income decile": labels_deciles,
    "Gain more than 5%": gain_more_than_5pct,
    "Gain less than 5%": gain_less_than_5pct,
    "No change": no_change,
    "Loss less than 5%": loss_less_than_5pct,
    "Loss more than 5%": loss_more_than_5pct
})

# Create DataFrame for All
df_all = pd.DataFrame({
    "Income decile": ["All"],
    "Gain more than 5%": [all_gain_more],
    "Gain less than 5%": [all_gain_less],
    "No change": [all_no_change],
    "Loss less than 5%": [all_loss_less],
    "Loss more than 5%": [all_loss_more]
})

# Create the final combined figure with a subplot
fig = make_subplots(
    rows=2,
    cols=1,
    shared_xaxes=True,
    vertical_spacing=0.02,
    row_heights=[0.1, 0.9]
)

# Add traces for "All" category - first row
fig.add_trace(go.Bar(
    y=df_all["Income decile"],
    x=df_all["Gain more than 5%"],
    name="Gain more than 5%",
    orientation='h',
    marker_color=BLUE_PRIMARY,
    text=[f"{x}%" if x > 0 else "" for x in df_all["Gain more than 5%"]],
    textposition="inside",
    legendgroup="Gain more than 5%",
    showlegend=True,
    hovertemplate="%{x}<extra></extra>"
), row=1, col=1)

fig.add_trace(go.Bar(
    y=df_all["Income decile"],
    x=df_all["Gain less than 5%"],
    name="Gain less than 5%",
    orientation='h',
    marker_color=BLUE_LIGHT,
    text=[f"{x}%" if x > 0 else "" for x in df_all["Gain less than 5%"]],
    textposition="inside",
    legendgroup="Gain less than 5%",
    showlegend=True,
    hovertemplate="%{x}<extra></extra>"
), row=1, col=1)

fig.add_trace(go.Bar(
    y=df_all["Income decile"],
    x=df_all["No change"],
    name="No change",
    orientation='h',
    marker_color=LIGHT_GRAY,
    text=[f"{x}%" if x > 0 else "" for x in df_all["No change"]],
    textposition="inside",
    legendgroup="No change",
    showlegend=True,
    hovertemplate="%{x}<extra></extra>"
), row=1, col=1)

fig.add_trace(go.Bar(
    y=df_all["Income decile"],
    x=df_all["Loss less than 5%"],
    name="Loss less than 5%",
    orientation='h',
    marker_color=MEDIUM_DARK_GRAY,
    text=[f"{x}%" if x > 0 else "" for x in df_all["Loss less than 5%"]],
    textposition="inside",
    legendgroup="Loss less than 5%",
    showlegend=True,
    hovertemplate="%{x}<extra></extra>"
), row=1, col=1)

fig.add_trace(go.Bar(
    y=df_all["Income decile"],
    x=df_all["Loss more than 5%"],
    name="Loss more than 5%",
    orientation='h',
    marker_color=DARK_GRAY,
    text=[f"{x}%" if x > 0 else "" for x in df_all["Loss more than 5%"]],
    textposition="inside",
    legendgroup="Loss more than 5%",
    showlegend=True,
    hovertemplate="%{x}<extra></extra>"
), row=1, col=1)

# Add traces for deciles - second row
fig.add_trace(go.Bar(
    y=df_deciles["Income decile"],
    x=df_deciles["Gain more than 5%"],
    name="Gain more than 5%",
    orientation='h',
    marker_color=BLUE_PRIMARY,
    text=[f"{x}%" if x > 0 else "" for x in df_deciles["Gain more than 5%"]],
    textposition="inside",
    legendgroup="Gain more than 5%",
    showlegend=False,
    hovertemplate="%{x}<extra></extra>"
), row=2, col=1)

fig.add_trace(go.Bar(
    y=df_deciles["Income decile"],
    x=df_deciles["Gain less than 5%"],
    name="Gain less than 5%",
    orientation='h',
    marker_color=BLUE_LIGHT,
    text=[f"{x}%" if x > 0 else "" for x in df_deciles["Gain less than 5%"]],
    textposition="inside",
    legendgroup="Gain less than 5%",
    showlegend=False,
    hovertemplate="%{x}<extra></extra>"
), row=2, col=1)

fig.add_trace(go.Bar(
    y=df_deciles["Income decile"],
    x=df_deciles["No change"],
    name="No change",
    orientation='h',
    marker_color=LIGHT_GRAY,
    text=[f"{x}%" if x > 0 else "" for x in df_deciles["No change"]],
    textposition="inside",
    legendgroup="No change",
    showlegend=False,
    hovertemplate="%{x}<extra></extra>"
), row=2, col=1)

fig.add_trace(go.Bar(
    y=df_deciles["Income decile"],
    x=df_deciles["Loss less than 5%"],
    name="Loss less than 5%",
    orientation='h',
    marker_color=MEDIUM_DARK_GRAY,
    text=[f"{x}%" if x > 0 else "" for x in df_deciles["Loss less than 5%"]],
    textposition="inside",
    legendgroup="Loss less than 5%",
    showlegend=False,
    hovertemplate="%{x}<extra></extra>"
), row=2, col=1)

fig.add_trace(go.Bar(
    y=df_deciles["Income decile"],
    x=df_deciles["Loss more than 5%"],
    name="Loss more than 5%",
    orientation='h',
    marker_color=DARK_GRAY,
    text=[f"{x}%" if x > 0 else "" for x in df_deciles["Loss more than 5%"]],
    textposition="inside",
    legendgroup="Loss more than 5%",
    showlegend=False,
    hovertemplate="%{x}<extra></extra>"
), row=2, col=1)

# Update layout
fig.update_layout(
    barmode='stack',
    title="Winners of Michigan's Income Tax Cut by Decile",
    title_x=0.8,
    font=dict(family="Roboto Serif"),
    xaxis=dict(title="", ticksuffix="%", range=[0, 100]),
    xaxis2=dict(
        title=dict(
            text="Population share",
            standoff=20  # Increase standoff to move title down
        ),
        ticksuffix="%",
        range=[0, 100]
    ),
    yaxis2=dict(
        automargin=True,
        title=dict(
            text="Income Decile",
            standoff=15  # Increase standoff to move title left
        )
    ),
    legend=dict(
        orientation="v", 
        yanchor="top",
        y=1.0,
        xanchor="right",
        x=1.5,
        traceorder="normal",
        itemsizing="constant"
    ),
    font_color=BLACK,
    margin={
      "l": 50,
      "r": 160,
      "b": 100,
      "t": 100,
      "pad": 4
    },
    height=600,
    width=900,  # Explicitly set width to accommodate legend
    annotations=[
      {
        "x": 1,
        "y": -0.25,
        "xref": "paper",
        "yref": "paper",
        "text": "Source: PolicyEngine US",
        "showarrow": False,
        "font": {
          "family": "Roboto Serif",
          "size": 10,
          "color": DARK_GRAY
        }
      }
    ],
    images=[
      {
        "source": "/logo512.png",
        "x": 1,
        "y": -0.2,
        "xref": "paper",
        "yref": "paper",
        "sizex": 0.1,
        "sizey": 0.1,
        "xanchor": "right",
        "yanchor": "bottom"
      }
    ],
)

# Display the figure
Markdown("**Figure 2: Winners of Idaho’s Income Tax Changes by Decile**\n\n" + f"```plotly\n{fig.to_json()}\n```")

**Figure 2: Winners of Idaho’s Income Tax Changes by Decile**

```plotly
{"data":[{"hovertemplate":"%{x}\u003cextra\u003e\u003c\u002fextra\u003e","legendgroup":"Gain more than 5%","marker":{"color":"#2C6496"},"name":"Gain more than 5%","orientation":"h","showlegend":true,"text":[""],"textposition":"inside","x":[0],"y":["All"],"type":"bar","xaxis":"x","yaxis":"y"},{"hovertemplate":"%{x}\u003cextra\u003e\u003c\u002fextra\u003e","legendgroup":"Gain less than 5%","marker":{"color":"#D8E6F3"},"name":"Gain less than 5%","orientation":"h","showlegend":true,"text":["87%"],"textposition":"inside","x":[87],"y":["All"],"type":"bar","xaxis":"x","yaxis":"y"},{"hovertemplate":"%{x}\u003cextra\u003e\u003c\u002fextra\u003e","legendgroup":"No change","marker":{"color":"#F2F2F2"},"name":"No change","orientation":"h","showlegend":true,"text":["13%"],"textposition":"inside","x":[13],"y":["All"],"type":"bar","xaxis":"x","yaxis":"y"},{"hovertemplate":"%{x}\u003cextra\u003e\u003c\u002fextra\u003e","legendgroup":"Loss less than 5%","marker":{"color":"#D2D2D2"},"name":"Loss less than 5%","orientation":"h","showlegend":true,"text":[""],"textposition":"inside","x":[0],"y":["All"],"type":"bar","xaxis":"x","yaxis":"y"},{"hovertemplate":"%{x}\u003cextra\u003e\u003c\u002fextra\u003e","legendgroup":"Loss more than 5%","marker":{"color":"#616161"},"name":"Loss more than 5%","orientation":"h","showlegend":true,"text":[""],"textposition":"inside","x":[0],"y":["All"],"type":"bar","xaxis":"x","yaxis":"y"},{"hovertemplate":"%{x}\u003cextra\u003e\u003c\u002fextra\u003e","legendgroup":"Gain more than 5%","marker":{"color":"#2C6496"},"name":"Gain more than 5%","orientation":"h","showlegend":false,"text":["2.4%","","","","","","","","",""],"textposition":"inside","x":[2.4,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0],"y":["1","2","3","4","5","6","7","8","9","10"],"type":"bar","xaxis":"x2","yaxis":"y2"},{"hovertemplate":"%{x}\u003cextra\u003e\u003c\u002fextra\u003e","legendgroup":"Gain less than 5%","marker":{"color":"#D8E6F3"},"name":"Gain less than 5%","orientation":"h","showlegend":false,"text":["43%","71%","82%","81%","95%","96%","100%","99%","100%","100%"],"textposition":"inside","x":[43,71,82,81,95,96,100,99,100,100],"y":["1","2","3","4","5","6","7","8","9","10"],"type":"bar","xaxis":"x2","yaxis":"y2"},{"hovertemplate":"%{x}\u003cextra\u003e\u003c\u002fextra\u003e","legendgroup":"No change","marker":{"color":"#F2F2F2"},"name":"No change","orientation":"h","showlegend":false,"text":["54%","29%","18%","19%","5%","4%","","1%","",""],"textposition":"inside","x":[54,29,18,19,5,4,0,1,0,0],"y":["1","2","3","4","5","6","7","8","9","10"],"type":"bar","xaxis":"x2","yaxis":"y2"},{"hovertemplate":"%{x}\u003cextra\u003e\u003c\u002fextra\u003e","legendgroup":"Loss less than 5%","marker":{"color":"#D2D2D2"},"name":"Loss less than 5%","orientation":"h","showlegend":false,"text":["","","","","","","","","",""],"textposition":"inside","x":[0,0,0,0,0,0,0,0,0,0],"y":["1","2","3","4","5","6","7","8","9","10"],"type":"bar","xaxis":"x2","yaxis":"y2"},{"hovertemplate":"%{x}\u003cextra\u003e\u003c\u002fextra\u003e","legendgroup":"Loss more than 5%","marker":{"color":"#616161"},"name":"Loss more than 5%","orientation":"h","showlegend":false,"text":["","","","","","","","","",""],"textposition":"inside","x":[0,0,0,0,0,0,0,0,0,0],"y":["1","2","3","4","5","6","7","8","9","10"],"type":"bar","xaxis":"x2","yaxis":"y2"}],"layout":{"template":{"data":{"histogram2dcontour":[{"type":"histogram2dcontour","colorbar":{"outlinewidth":0,"ticks":""},"colorscale":[[0.0,"#0d0887"],[0.1111111111111111,"#46039f"],[0.2222222222222222,"#7201a8"],[0.3333333333333333,"#9c179e"],[0.4444444444444444,"#bd3786"],[0.5555555555555556,"#d8576b"],[0.6666666666666666,"#ed7953"],[0.7777777777777778,"#fb9f3a"],[0.8888888888888888,"#fdca26"],[1.0,"#f0f921"]]}],"choropleth":[{"type":"choropleth","colorbar":{"outlinewidth":0,"ticks":""}}],"histogram2d":[{"type":"histogram2d","colorbar":{"outlinewidth":0,"ticks":""},"colorscale":[[0.0,"#0d0887"],[0.1111111111111111,"#46039f"],[0.2222222222222222,"#7201a8"],[0.3333333333333333,"#9c179e"],[0.4444444444444444,"#bd3786"],[0.5555555555555556,"#d8576b"],[0.6666666666666666,"#ed7953"],[0.7777777777777778,"#fb9f3a"],[0.8888888888888888,"#fdca26"],[1.0,"#f0f921"]]}],"heatmap":[{"type":"heatmap","colorbar":{"outlinewidth":0,"ticks":""},"colorscale":[[0.0,"#0d0887"],[0.1111111111111111,"#46039f"],[0.2222222222222222,"#7201a8"],[0.3333333333333333,"#9c179e"],[0.4444444444444444,"#bd3786"],[0.5555555555555556,"#d8576b"],[0.6666666666666666,"#ed7953"],[0.7777777777777778,"#fb9f3a"],[0.8888888888888888,"#fdca26"],[1.0,"#f0f921"]]}],"heatmapgl":[{"type":"heatmapgl","colorbar":{"outlinewidth":0,"ticks":""},"colorscale":[[0.0,"#0d0887"],[0.1111111111111111,"#46039f"],[0.2222222222222222,"#7201a8"],[0.3333333333333333,"#9c179e"],[0.4444444444444444,"#bd3786"],[0.5555555555555556,"#d8576b"],[0.6666666666666666,"#ed7953"],[0.7777777777777778,"#fb9f3a"],[0.8888888888888888,"#fdca26"],[1.0,"#f0f921"]]}],"contourcarpet":[{"type":"contourcarpet","colorbar":{"outlinewidth":0,"ticks":""}}],"contour":[{"type":"contour","colorbar":{"outlinewidth":0,"ticks":""},"colorscale":[[0.0,"#0d0887"],[0.1111111111111111,"#46039f"],[0.2222222222222222,"#7201a8"],[0.3333333333333333,"#9c179e"],[0.4444444444444444,"#bd3786"],[0.5555555555555556,"#d8576b"],[0.6666666666666666,"#ed7953"],[0.7777777777777778,"#fb9f3a"],[0.8888888888888888,"#fdca26"],[1.0,"#f0f921"]]}],"surface":[{"type":"surface","colorbar":{"outlinewidth":0,"ticks":""},"colorscale":[[0.0,"#0d0887"],[0.1111111111111111,"#46039f"],[0.2222222222222222,"#7201a8"],[0.3333333333333333,"#9c179e"],[0.4444444444444444,"#bd3786"],[0.5555555555555556,"#d8576b"],[0.6666666666666666,"#ed7953"],[0.7777777777777778,"#fb9f3a"],[0.8888888888888888,"#fdca26"],[1.0,"#f0f921"]]}],"mesh3d":[{"type":"mesh3d","colorbar":{"outlinewidth":0,"ticks":""}}],"scatter":[{"fillpattern":{"fillmode":"overlay","size":10,"solidity":0.2},"type":"scatter"}],"parcoords":[{"type":"parcoords","line":{"colorbar":{"outlinewidth":0,"ticks":""}}}],"scatterpolargl":[{"type":"scatterpolargl","marker":{"colorbar":{"outlinewidth":0,"ticks":""}}}],"bar":[{"error_x":{"color":"#2a3f5f"},"error_y":{"color":"#2a3f5f"},"marker":{"line":{"color":"#E5ECF6","width":0.5},"pattern":{"fillmode":"overlay","size":10,"solidity":0.2}},"type":"bar"}],"scattergeo":[{"type":"scattergeo","marker":{"colorbar":{"outlinewidth":0,"ticks":""}}}],"scatterpolar":[{"type":"scatterpolar","marker":{"colorbar":{"outlinewidth":0,"ticks":""}}}],"histogram":[{"marker":{"pattern":{"fillmode":"overlay","size":10,"solidity":0.2}},"type":"histogram"}],"scattergl":[{"type":"scattergl","marker":{"colorbar":{"outlinewidth":0,"ticks":""}}}],"scatter3d":[{"type":"scatter3d","line":{"colorbar":{"outlinewidth":0,"ticks":""}},"marker":{"colorbar":{"outlinewidth":0,"ticks":""}}}],"scattermapbox":[{"type":"scattermapbox","marker":{"colorbar":{"outlinewidth":0,"ticks":""}}}],"scatterternary":[{"type":"scatterternary","marker":{"colorbar":{"outlinewidth":0,"ticks":""}}}],"scattercarpet":[{"type":"scattercarpet","marker":{"colorbar":{"outlinewidth":0,"ticks":""}}}],"carpet":[{"aaxis":{"endlinecolor":"#2a3f5f","gridcolor":"white","linecolor":"white","minorgridcolor":"white","startlinecolor":"#2a3f5f"},"baxis":{"endlinecolor":"#2a3f5f","gridcolor":"white","linecolor":"white","minorgridcolor":"white","startlinecolor":"#2a3f5f"},"type":"carpet"}],"table":[{"cells":{"fill":{"color":"#EBF0F8"},"line":{"color":"white"}},"header":{"fill":{"color":"#C8D4E3"},"line":{"color":"white"}},"type":"table"}],"barpolar":[{"marker":{"line":{"color":"#E5ECF6","width":0.5},"pattern":{"fillmode":"overlay","size":10,"solidity":0.2}},"type":"barpolar"}],"pie":[{"automargin":true,"type":"pie"}]},"layout":{"autotypenumbers":"strict","colorway":["#636efa","#EF553B","#00cc96","#ab63fa","#FFA15A","#19d3f3","#FF6692","#B6E880","#FF97FF","#FECB52"],"font":{"color":"#2a3f5f"},"hovermode":"closest","hoverlabel":{"align":"left"},"paper_bgcolor":"white","plot_bgcolor":"#E5ECF6","polar":{"bgcolor":"#E5ECF6","angularaxis":{"gridcolor":"white","linecolor":"white","ticks":""},"radialaxis":{"gridcolor":"white","linecolor":"white","ticks":""}},"ternary":{"bgcolor":"#E5ECF6","aaxis":{"gridcolor":"white","linecolor":"white","ticks":""},"baxis":{"gridcolor":"white","linecolor":"white","ticks":""},"caxis":{"gridcolor":"white","linecolor":"white","ticks":""}},"coloraxis":{"colorbar":{"outlinewidth":0,"ticks":""}},"colorscale":{"sequential":[[0.0,"#0d0887"],[0.1111111111111111,"#46039f"],[0.2222222222222222,"#7201a8"],[0.3333333333333333,"#9c179e"],[0.4444444444444444,"#bd3786"],[0.5555555555555556,"#d8576b"],[0.6666666666666666,"#ed7953"],[0.7777777777777778,"#fb9f3a"],[0.8888888888888888,"#fdca26"],[1.0,"#f0f921"]],"sequentialminus":[[0.0,"#0d0887"],[0.1111111111111111,"#46039f"],[0.2222222222222222,"#7201a8"],[0.3333333333333333,"#9c179e"],[0.4444444444444444,"#bd3786"],[0.5555555555555556,"#d8576b"],[0.6666666666666666,"#ed7953"],[0.7777777777777778,"#fb9f3a"],[0.8888888888888888,"#fdca26"],[1.0,"#f0f921"]],"diverging":[[0,"#8e0152"],[0.1,"#c51b7d"],[0.2,"#de77ae"],[0.3,"#f1b6da"],[0.4,"#fde0ef"],[0.5,"#f7f7f7"],[0.6,"#e6f5d0"],[0.7,"#b8e186"],[0.8,"#7fbc41"],[0.9,"#4d9221"],[1,"#276419"]]},"xaxis":{"gridcolor":"white","linecolor":"white","ticks":"","title":{"standoff":15},"zerolinecolor":"white","automargin":true,"zerolinewidth":2},"yaxis":{"gridcolor":"white","linecolor":"white","ticks":"","title":{"standoff":15},"zerolinecolor":"white","automargin":true,"zerolinewidth":2},"scene":{"xaxis":{"backgroundcolor":"#E5ECF6","gridcolor":"white","linecolor":"white","showbackground":true,"ticks":"","zerolinecolor":"white","gridwidth":2},"yaxis":{"backgroundcolor":"#E5ECF6","gridcolor":"white","linecolor":"white","showbackground":true,"ticks":"","zerolinecolor":"white","gridwidth":2},"zaxis":{"backgroundcolor":"#E5ECF6","gridcolor":"white","linecolor":"white","showbackground":true,"ticks":"","zerolinecolor":"white","gridwidth":2}},"shapedefaults":{"line":{"color":"#2a3f5f"}},"annotationdefaults":{"arrowcolor":"#2a3f5f","arrowhead":0,"arrowwidth":1},"geo":{"bgcolor":"white","landcolor":"#E5ECF6","subunitcolor":"white","showland":true,"showlakes":true,"lakecolor":"white"},"title":{"x":0.05},"mapbox":{"style":"light"}}},"xaxis":{"anchor":"y","domain":[0.0,1.0],"matches":"x2","showticklabels":false,"title":{"text":""},"ticksuffix":"%","range":[0,100]},"yaxis":{"anchor":"x","domain":[0.902,1.0]},"xaxis2":{"anchor":"y2","domain":[0.0,1.0],"title":{"text":"Population share","standoff":20},"ticksuffix":"%","range":[0,100]},"yaxis2":{"anchor":"x2","domain":[0.0,0.882],"title":{"text":"Income Decile","standoff":15},"automargin":true},"title":{"text":"Winners of Michigan's Income Tax Cut by Decile","x":0.8},"font":{"family":"Roboto Serif","color":"#000000"},"legend":{"orientation":"v","yanchor":"top","y":1.0,"xanchor":"right","x":1.5,"traceorder":"normal","itemsizing":"constant"},"margin":{"l":50,"r":160,"b":100,"t":100,"pad":4},"barmode":"stack","height":600,"width":900,"annotations":[{"font":{"color":"#616161","family":"Roboto Serif","size":10},"showarrow":false,"text":"Source: PolicyEngine US","x":1,"xref":"paper","y":-0.25,"yref":"paper"}],"images":[{"sizex":0.1,"sizey":0.1,"source":"\u002flogo512.png","x":1,"xanchor":"right","xref":"paper","y":-0.2,"yanchor":"bottom","yref":"paper"}]}}
```

The tax package will provide an [average benefit of $292 per household](https://policyengine.org/us/policy?focus=policyOutput.distributionalImpact.incomeDecile.average&reform=2&region=id&timePeriod=2025&baseline=80392&household=52252), ranging from $25 in the bottom income decile to $1,193 in the top decile (as defined by the nationwide income distribution).

In [7]:
# Average impact by decile data
deciles = list(range(1, 11))
avg_impact = [25, 74, 120, 151, 250, 300, 365, 454, 602, 1193]

df = pd.DataFrame({
    "Income decile": deciles,
    "Average impact": avg_impact,
})

# Format dollar values for display
dollar_text = [f"${x}" for x in avg_impact]

fig = px.bar(
    df,
    x="Income decile",
    y="Average impact",
    text=dollar_text,  # Dollar values with $ sign
    color_discrete_sequence=[BLUE_PRIMARY],
    title="Income Distribution of Michigan's Income Tax Cut by Decile",
).update_layout(
    font=dict(family="Roboto Serif"),
    xaxis_title="Income decile",
    yaxis_title="Average impact ($)",
    xaxis_tickvals=list(range(1, 11)),
    yaxis_tickformat=",",
    showlegend=False,
    font_color=BLACK,
    margin={
      "l": 50,
      "r": 50,
      "b": 100,
      "t": 100,
      "pad": 4
    },
    annotations=[
      {
        "x": 1,
        "y": -0.25,
        "xref": "paper",
        "yref": "paper",
        "text": "Source: PolicyEngine US",
        "showarrow": False,
        "font": {
          "family": "Roboto Serif",
          "size": 10,
          "color": DARK_GRAY
        }
      }
    ],
    images=[
      {
        "source": "/logo512.png",
        "x": 1,
        "y": -0.2,
        "xref": "paper",
        "yref": "paper",
        "sizex": 0.1,
        "sizey": 0.1,
        "xanchor": "right",
        "yanchor": "bottom"
      }
    ],
).update_traces(
    hovertemplate="Income decile: %{x}<br>Average impact: $%{y:,.0f}<extra></extra>"
)

Markdown("**Figure 3: Average Benefit of Idaho’s Income Tax Changes by Decile**\n\n" + f"```plotly\n{fig.to_json()}\n```")

**Figure 3: Average Benefit of Idaho’s Income Tax Changes by Decile**

```plotly
{"data":[{"alignmentgroup":"True","hovertemplate":"Income decile: %{x}\u003cbr\u003eAverage impact: $%{y:,.0f}\u003cextra\u003e\u003c\u002fextra\u003e","legendgroup":"","marker":{"color":"#2C6496","pattern":{"shape":""}},"name":"","offsetgroup":"","orientation":"v","showlegend":false,"text":["$25","$74","$120","$151","$250","$300","$365","$454","$602","$1193"],"textposition":"auto","x":[1,2,3,4,5,6,7,8,9,10],"xaxis":"x","y":[25,74,120,151,250,300,365,454,602,1193],"yaxis":"y","type":"bar"}],"layout":{"template":{"data":{"histogram2dcontour":[{"type":"histogram2dcontour","colorbar":{"outlinewidth":0,"ticks":""},"colorscale":[[0.0,"#0d0887"],[0.1111111111111111,"#46039f"],[0.2222222222222222,"#7201a8"],[0.3333333333333333,"#9c179e"],[0.4444444444444444,"#bd3786"],[0.5555555555555556,"#d8576b"],[0.6666666666666666,"#ed7953"],[0.7777777777777778,"#fb9f3a"],[0.8888888888888888,"#fdca26"],[1.0,"#f0f921"]]}],"choropleth":[{"type":"choropleth","colorbar":{"outlinewidth":0,"ticks":""}}],"histogram2d":[{"type":"histogram2d","colorbar":{"outlinewidth":0,"ticks":""},"colorscale":[[0.0,"#0d0887"],[0.1111111111111111,"#46039f"],[0.2222222222222222,"#7201a8"],[0.3333333333333333,"#9c179e"],[0.4444444444444444,"#bd3786"],[0.5555555555555556,"#d8576b"],[0.6666666666666666,"#ed7953"],[0.7777777777777778,"#fb9f3a"],[0.8888888888888888,"#fdca26"],[1.0,"#f0f921"]]}],"heatmap":[{"type":"heatmap","colorbar":{"outlinewidth":0,"ticks":""},"colorscale":[[0.0,"#0d0887"],[0.1111111111111111,"#46039f"],[0.2222222222222222,"#7201a8"],[0.3333333333333333,"#9c179e"],[0.4444444444444444,"#bd3786"],[0.5555555555555556,"#d8576b"],[0.6666666666666666,"#ed7953"],[0.7777777777777778,"#fb9f3a"],[0.8888888888888888,"#fdca26"],[1.0,"#f0f921"]]}],"heatmapgl":[{"type":"heatmapgl","colorbar":{"outlinewidth":0,"ticks":""},"colorscale":[[0.0,"#0d0887"],[0.1111111111111111,"#46039f"],[0.2222222222222222,"#7201a8"],[0.3333333333333333,"#9c179e"],[0.4444444444444444,"#bd3786"],[0.5555555555555556,"#d8576b"],[0.6666666666666666,"#ed7953"],[0.7777777777777778,"#fb9f3a"],[0.8888888888888888,"#fdca26"],[1.0,"#f0f921"]]}],"contourcarpet":[{"type":"contourcarpet","colorbar":{"outlinewidth":0,"ticks":""}}],"contour":[{"type":"contour","colorbar":{"outlinewidth":0,"ticks":""},"colorscale":[[0.0,"#0d0887"],[0.1111111111111111,"#46039f"],[0.2222222222222222,"#7201a8"],[0.3333333333333333,"#9c179e"],[0.4444444444444444,"#bd3786"],[0.5555555555555556,"#d8576b"],[0.6666666666666666,"#ed7953"],[0.7777777777777778,"#fb9f3a"],[0.8888888888888888,"#fdca26"],[1.0,"#f0f921"]]}],"surface":[{"type":"surface","colorbar":{"outlinewidth":0,"ticks":""},"colorscale":[[0.0,"#0d0887"],[0.1111111111111111,"#46039f"],[0.2222222222222222,"#7201a8"],[0.3333333333333333,"#9c179e"],[0.4444444444444444,"#bd3786"],[0.5555555555555556,"#d8576b"],[0.6666666666666666,"#ed7953"],[0.7777777777777778,"#fb9f3a"],[0.8888888888888888,"#fdca26"],[1.0,"#f0f921"]]}],"mesh3d":[{"type":"mesh3d","colorbar":{"outlinewidth":0,"ticks":""}}],"scatter":[{"fillpattern":{"fillmode":"overlay","size":10,"solidity":0.2},"type":"scatter"}],"parcoords":[{"type":"parcoords","line":{"colorbar":{"outlinewidth":0,"ticks":""}}}],"scatterpolargl":[{"type":"scatterpolargl","marker":{"colorbar":{"outlinewidth":0,"ticks":""}}}],"bar":[{"error_x":{"color":"#2a3f5f"},"error_y":{"color":"#2a3f5f"},"marker":{"line":{"color":"#E5ECF6","width":0.5},"pattern":{"fillmode":"overlay","size":10,"solidity":0.2}},"type":"bar"}],"scattergeo":[{"type":"scattergeo","marker":{"colorbar":{"outlinewidth":0,"ticks":""}}}],"scatterpolar":[{"type":"scatterpolar","marker":{"colorbar":{"outlinewidth":0,"ticks":""}}}],"histogram":[{"marker":{"pattern":{"fillmode":"overlay","size":10,"solidity":0.2}},"type":"histogram"}],"scattergl":[{"type":"scattergl","marker":{"colorbar":{"outlinewidth":0,"ticks":""}}}],"scatter3d":[{"type":"scatter3d","line":{"colorbar":{"outlinewidth":0,"ticks":""}},"marker":{"colorbar":{"outlinewidth":0,"ticks":""}}}],"scattermapbox":[{"type":"scattermapbox","marker":{"colorbar":{"outlinewidth":0,"ticks":""}}}],"scatterternary":[{"type":"scatterternary","marker":{"colorbar":{"outlinewidth":0,"ticks":""}}}],"scattercarpet":[{"type":"scattercarpet","marker":{"colorbar":{"outlinewidth":0,"ticks":""}}}],"carpet":[{"aaxis":{"endlinecolor":"#2a3f5f","gridcolor":"white","linecolor":"white","minorgridcolor":"white","startlinecolor":"#2a3f5f"},"baxis":{"endlinecolor":"#2a3f5f","gridcolor":"white","linecolor":"white","minorgridcolor":"white","startlinecolor":"#2a3f5f"},"type":"carpet"}],"table":[{"cells":{"fill":{"color":"#EBF0F8"},"line":{"color":"white"}},"header":{"fill":{"color":"#C8D4E3"},"line":{"color":"white"}},"type":"table"}],"barpolar":[{"marker":{"line":{"color":"#E5ECF6","width":0.5},"pattern":{"fillmode":"overlay","size":10,"solidity":0.2}},"type":"barpolar"}],"pie":[{"automargin":true,"type":"pie"}]},"layout":{"autotypenumbers":"strict","colorway":["#636efa","#EF553B","#00cc96","#ab63fa","#FFA15A","#19d3f3","#FF6692","#B6E880","#FF97FF","#FECB52"],"font":{"color":"#2a3f5f"},"hovermode":"closest","hoverlabel":{"align":"left"},"paper_bgcolor":"white","plot_bgcolor":"#E5ECF6","polar":{"bgcolor":"#E5ECF6","angularaxis":{"gridcolor":"white","linecolor":"white","ticks":""},"radialaxis":{"gridcolor":"white","linecolor":"white","ticks":""}},"ternary":{"bgcolor":"#E5ECF6","aaxis":{"gridcolor":"white","linecolor":"white","ticks":""},"baxis":{"gridcolor":"white","linecolor":"white","ticks":""},"caxis":{"gridcolor":"white","linecolor":"white","ticks":""}},"coloraxis":{"colorbar":{"outlinewidth":0,"ticks":""}},"colorscale":{"sequential":[[0.0,"#0d0887"],[0.1111111111111111,"#46039f"],[0.2222222222222222,"#7201a8"],[0.3333333333333333,"#9c179e"],[0.4444444444444444,"#bd3786"],[0.5555555555555556,"#d8576b"],[0.6666666666666666,"#ed7953"],[0.7777777777777778,"#fb9f3a"],[0.8888888888888888,"#fdca26"],[1.0,"#f0f921"]],"sequentialminus":[[0.0,"#0d0887"],[0.1111111111111111,"#46039f"],[0.2222222222222222,"#7201a8"],[0.3333333333333333,"#9c179e"],[0.4444444444444444,"#bd3786"],[0.5555555555555556,"#d8576b"],[0.6666666666666666,"#ed7953"],[0.7777777777777778,"#fb9f3a"],[0.8888888888888888,"#fdca26"],[1.0,"#f0f921"]],"diverging":[[0,"#8e0152"],[0.1,"#c51b7d"],[0.2,"#de77ae"],[0.3,"#f1b6da"],[0.4,"#fde0ef"],[0.5,"#f7f7f7"],[0.6,"#e6f5d0"],[0.7,"#b8e186"],[0.8,"#7fbc41"],[0.9,"#4d9221"],[1,"#276419"]]},"xaxis":{"gridcolor":"white","linecolor":"white","ticks":"","title":{"standoff":15},"zerolinecolor":"white","automargin":true,"zerolinewidth":2},"yaxis":{"gridcolor":"white","linecolor":"white","ticks":"","title":{"standoff":15},"zerolinecolor":"white","automargin":true,"zerolinewidth":2},"scene":{"xaxis":{"backgroundcolor":"#E5ECF6","gridcolor":"white","linecolor":"white","showbackground":true,"ticks":"","zerolinecolor":"white","gridwidth":2},"yaxis":{"backgroundcolor":"#E5ECF6","gridcolor":"white","linecolor":"white","showbackground":true,"ticks":"","zerolinecolor":"white","gridwidth":2},"zaxis":{"backgroundcolor":"#E5ECF6","gridcolor":"white","linecolor":"white","showbackground":true,"ticks":"","zerolinecolor":"white","gridwidth":2}},"shapedefaults":{"line":{"color":"#2a3f5f"}},"annotationdefaults":{"arrowcolor":"#2a3f5f","arrowhead":0,"arrowwidth":1},"geo":{"bgcolor":"white","landcolor":"#E5ECF6","subunitcolor":"white","showland":true,"showlakes":true,"lakecolor":"white"},"title":{"x":0.05},"mapbox":{"style":"light"}}},"xaxis":{"anchor":"y","domain":[0.0,1.0],"title":{"text":"Income decile"},"tickvals":[1,2,3,4,5,6,7,8,9,10]},"yaxis":{"anchor":"x","domain":[0.0,1.0],"title":{"text":"Average impact ($)"},"tickformat":","},"legend":{"tracegroupgap":0},"title":{"text":"Income Distribution of Michigan's Income Tax Cut by Decile"},"barmode":"relative","font":{"family":"Roboto Serif","color":"#000000"},"margin":{"l":50,"r":50,"b":100,"t":100,"pad":4},"showlegend":false,"annotations":[{"font":{"color":"#616161","family":"Roboto Serif","size":10},"showarrow":false,"text":"Source: PolicyEngine US","x":1,"xref":"paper","y":-0.25,"yref":"paper"}],"images":[{"sizex":0.1,"sizey":0.1,"source":"\u002flogo512.png","x":1,"xanchor":"right","xref":"paper","y":-0.2,"yanchor":"bottom","yref":"paper"}]}}
```

The tax changes are projected to have [no effect on poverty](https://policyengine.org/us/policy?focus=policyOutput.povertyImpact.regular.byAge&reform=2&region=id&timePeriod=2025&baseline=80392&household=52252) or deep poverty while increasing the state’s Gini index of inequality [by 0.01%](https://policyengine.org/us/policy?focus=policyOutput.inequalityImpact&reform=2&region=id&timePeriod=2025&baseline=80392&household=52252). 

## Conclusion

The analyzed individual income tax changes from Idaho’s 2025 legislative session will lower state revenues by $224.9 million in 2025, or $292 per household. Households in higher income deciles will benefit more than those in lower income deciles on an [absolute](https://policyengine.org/us/policy?focus=policyOutput.distributionalImpact.incomeDecile.average&reform=2&region=id&timePeriod=2025&baseline=80392&household=52252) and [relative](https://policyengine.org/us/policy?focus=policyOutput.distributionalImpact.incomeDecile.relative&reform=2&region=id&timePeriod=2025&baseline=80392&household=52252) basis. The bill will not affect the state’s supplemental poverty measure and increase the Gini index of income inequality by 0.01%. 

As policymakers evaluate reforms such as these, analytical tools like PolicyEngine offer critical insights into the impacts on diverse household compositions and the broader economy.

We invite you to explore our [additional analyses](https://policyengine.org/us/research) and use [PolicyEngine](https://policyengine.org/us) to calculate your own tax benefits or design custom policy reforms.